WooCommerce/Classes/ViewRelated/Products/Variations/Add Attributes/AddAttributeOptionsViewController.swift

```diff
@@ -1,5 +1,6 @@
 import UIKit
 import Yosemite
+import WordPressUI
```

Comment:
Do we really need this import? I removed it and the project seems to build as expected

Reply:
Forgot to remove it! Good catch!